src/modules/InstURL.rb

```diff
@@ -142,232 +83,68 @@ def RewriteCDUrl(url)
-        if instmode == "smb" && share != nil && share != ""
-          Ops.set(url_tokens, "path", Ops.add(share, "/"))
-        else
-          Ops.set(url_tokens, "path", "/")
-        end
-      end
-
-      port = Linuxrc.InstallInf("Port")
-      Ops.set(url_tokens, "port", port) if port != nil && port != ""
-
-      url = URL.Build(url_tokens)
-      option_separator = "?"
-
-      if @is_network
-        proxy = Linuxrc.InstallInf("Proxy")
-        if proxy != nil && proxy != ""
-          url = Ops.add(
-            Ops.add(Ops.add(url, option_separator), "proxy="),
-            proxy
-          )
-          option_separator = "&"
-        end
-        proxyport = Linuxrc.InstallInf("ProxyPort")
-        if proxyport != nil && proxyport != ""
-          url = Ops.add(
-            Ops.add(Ops.add(url, option_separator), "proxyport="),
-            proxyport
-          )
-          option_separator = "&"
-        end
-        proxyproto = Linuxrc.InstallInf("ProxyProto")
-        if proxyproto != nil && proxyproto != ""
-          url = Ops.add(
-            Ops.add(Ops.add(url, option_separator), "proxyproto="),
-            proxyproto
-          )
-          option_separator = "&"
-        end
-        proxyuser = Linuxrc.InstallInf("ProxyUser")
-        if proxyuser != nil && proxyuser != ""
-          url = Ops.add(
-            Ops.add(Ops.add(url, option_separator), "proxyuser="),
-            proxyuser
-          )
-          option_separator = "&"
-        end
-        proxypassword = Linuxrc.InstallInf("ProxyPassword")
-        if proxypassword != nil && proxypassword != ""
-          url = Ops.add(
-            Ops.add(Ops.add(url, option_separator), "proxypassword="),
-            proxypassword
-          )
-          option_separator = "&"
-        end
-        workgroup = Linuxrc.InstallInf("WorkDomain")
-        if workgroup != nil && workgroup != ""
-          url = Ops.add(
-            Ops.add(Ops.add(url, option_separator), "workgroup="),
-            workgroup
-          )
-          option_separator = "&"
-        end
-
-        if instmode == "https"
-          Builtins.y2milestone("HTTPS instmode detected")
+      log.info "Using install URL: #{URL.HidePassword(@installInf2Url)}"
+      @installInf2Url
+    end
 
-          if !SSLVerificationEnabled()
-            Builtins.y2security(
-              "Disabling certificate check for the installation repository"
-            )
+    # Schemes considered local for installInf2Url
+    LOCAL_SCHEMES = ["cd", "dvd", "hd"]
 
-            # libzypp uses ssl_verify=no option
-            url = Ops.add(Ops.add(url, option_separator), "ssl_verify=no")
-            option_separator = "&"
-          end
-        end
-      end # is_network
+    # Determines whether the installation URL is remote or not
+    #
+    # @return [Boolean] true if it's remote; false otherwise.
+    # @see installInf2Url
+    def is_network
```

Comment:
I'm fine with dropping it. I just wanted to keep the API but I'm pretty sure that it's not used anymore. And about 3th party YaST modules I would be suprissed if it's used.